misc/testing/Release/resetSearchname.php

```diff
@@ -19,7 +19,7 @@
 }
 
 if (isset($argv[1]) && $argv[1] == "full") {
-	$res = $pdo->query("SELECT releases.id, releases.name, releases.fromname, releases.size, groups.name AS gname FROM releases INNER JOIN groups ON releases.group_id = groups.id");
+	$res = $pdo->query("SELECT releases.id, releases.name, releases.fromname, releases.size, groups.name AS gname FROM releases INNER JOIN groups ON releases.groups_id = groups.id");
```

Comment:
We should probably standardize the formatting of these and use table aliases instead of table.column format.